app/src/components/fields/MultiAutocompleteField.tsx

```diff
@@ -93,45 +103,46 @@ const MultiAutocompleteField: React.FC<IMultiAutocompleteField> = (props) => {
     }
   };
 
-  const defaultChipDisplay = (option: any, renderProps: any, checkedStatus: any) => {
-    return (
-      <Box component="li" {...renderProps}>
-        <Checkbox
-          icon={<CheckBoxOutlineBlank fontSize="small" />}
-          checkedIcon={<CheckBox fontSize="small" color="primary" />}
-          style={{ marginRight: 8 }}
-          checked={checkedStatus}
-          disabled={(options && options?.indexOf(option) !== -1) || false}
-          value={option.value}
-          color="default"
-        />
-        {option.label}
-      </Box>
-    );
+  const getExistingValue = (existingValues: (number | string)[]): IMultiAutocompleteFieldOption[] => {
+    if (existingValues) {
+      return options.filter((option) => existingValues.includes(option.value));
+    }
+    return [];
   };
 
-  const existingValues: IMultiAutocompleteFieldOption[] =
-    value && value.length > 0 ? options.filter((option) => value.includes(option)) : [];
   return (
     <Autocomplete
       multiple
+      noOptionsText="No matching options"
       autoHighlight={true}
-      value={existingValues}
+      value={getExistingValue(get(values, props.id))}
       id={props.id}
+      data-testid={props.id}
       options={options}
       getOptionLabel={(option) => option.label}
       isOptionEqualToValue={handleGetOptionSelected}
-      filterOptions={createFilterOptions({ limit: props.filterLimit })}
       disableCloseOnSelect
-      onChange={handleOnChange}
+      disableListWrap
       inputValue={inputValue}
       onInputChange={handleOnInputChange}
-      renderTags={(tagValue, getTagProps) => {
-        if (props.chipVisible) {
-          return tagValue.map((option, index) => <Chip label={option.label} {...getTagProps({ index })} />);
-        }
+      onChange={props.onChange ? props.onChange : defaultHandleOnChange}
+      filterOptions={createFilterOptions({ limit: props.filterLimit })}
+      renderOption={(renderProps, renderOption, { selected }) => {
+        return (
+          <Box component="li" {...renderProps}>
+            <Checkbox
+              icon={<CheckBoxOutlineBlank fontSize="small" />}
+              checkedIcon={<CheckBox fontSize="small" />}
+              checked={selected}
+              // Always seem to be disabled
+              disabled={(props.options && props.options?.indexOf(renderOption) !== -1) || false}
```

Comment:
Could probably use `props.options?.includes(renderOption)`

Reply:
For sure

Though tbh I am not really sure what this is even doing. Its trying to grey out options that aren't in the list of options, or something... Seems like an odd use case that we probably never actually encounter in our use cases. Probably just something copy/pasted from an example somewhere.